HowWeWork/CompensationStructure.md

```diff
@@ -47,17 +47,23 @@ Rather than focus on "experience" or "tenure", we prefer to give our team member
 - **Vacation** - You do not need to ask permission to take time off. Always make sure that your job responsibilities are covered while you are away. We strongly recommended to take at least a minimum of 2 weeks of vacation per year, if you take less, we might follow up to review your work load.
 
 - **Checking Vacation Calendars** - You do need to ensure that not more than half of the people that can help with availability emergencies (the on-call heroes), regular support, sales, or development are gone at any moment. You can check for this on the availability calendar, so be sure to add appointments early. Add an appointment to the Walkthrough availability calendar (Internal Document) as you know your plans, you can always change it later.
-<br><br><br><br>
+
+- **Working Hours** - Although team members can work any hours of their choosing, we ask that everyone be available for meetings anytime between 10am - 3:00pm, Mondays through and including Fridays.
```

Comment:
We need to include at timezone here. I think we can just say between 10am and 3pm Mountain Time. I would also say we should be more general and say "we ask that everyone's normal schedule include availability for meetings and for collaborating with co-workers between..."